docs/rules/v-on-handler-style.md

```diff
@@ -106,50 +149,52 @@ This rule aims to enforce a consistent style in `v-on` event handlers:
   <!-- ✓ GOOD -->
   <button v-on:click="handler()" />
   <button v-on:click="handler($event)" />
+  <button v-on:click="count++" />
 
   <!-- ✗ BAD -->
   <button v-on:click="handler" />
   <button v-on:click="() => handler()" />
   <button v-on:click="(foo) => handler(foo)" />
-
-  <!-- Ignore -->
   <button v-on:click="(foo, bar) => handler(foo, bar)" />
+  <button v-on:click="() => count++" />
 </template>
 ```
 
 </eslint-code-block>
 
-### `["inline-function"]`
+### `["method", "inline-function"], { "ignoreIncludesComment": true }`
 
-<eslint-code-block fix :rules="{'vue/v-on-handler-style': ['error', ['inline-function']]}">
+<eslint-code-block fix :rules="{'vue/v-on-handler-style': ['error', ['method', 'inline-function'], {ignoreIncludesComment: true}]}">
 
 ```vue
 <template>
   <!-- ✓ GOOD -->
-  <button v-on:click="() => handler()" />
+  <button v-on:click="handler" />
+  <button v-on:click="() => handler() /* comment */" />
 
   <!-- ✗ BAD -->
-  <button v-on:click="handler" />
   <button v-on:click="handler()" />
+  <button v-on:click="() => handler()" />
+  <button v-on:click="handler() /* comment */" />
```

Comment:
I think with `ignoreIncludesComment: true`, this should be also allowed?

Reply:
The `ignoreIncludesComment` option is only ignored when converting to method handlers. Because the user can't convert it to a method handler without removing the comment.
Comments can be kept between inline handler <-> inline function.
I wrote an option description but I can't explain it well 😓

Reply:
Ah, I see. Can you check if my explanation suggestion below is correct?

Reply:
Thank you for your suggestion! I have accepted it into this PR!